conftest.py

```diff
@@ -84,7 +84,7 @@ def pytest_collection_modifyitems(config, items):
         ]
 
         if os.name == 'nt':
-            percentage = 0.5
+            percentage = 0.2
```

Comment:
```suggestion
            percentage = 0.4
```

Let's change this to `0.4` to see if the slow and fast slots on Windows use about the same time.